go_router/lib/src/logging.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+import 'dart:developer' as developer;
+import 'package:logging/logging.dart';
+
+/// The logger for this package.
+Logger log = Logger('GoRouter');
+
+StreamSubscription? _subscription;
+
+/// Forwards diagnostic messages to the dart:developer log() API.
+void setLogging({bool enabled = false}) {
+  _subscription?.cancel();
+  if (!enabled) {
+    return;
+  }
+
+  _subscription = Logger.root.onRecord.listen((e) {
```

Comment:
@pq is there a better way to forward logs to `developer.log()`?

Reply:
Not that I'm aware of...

@devoncarew might have ideas?

Reply:
No, I think is you want to forward from package:logging to developer log(), this is the way to do it.